### PR TITLE
Increases the accuracy of the photocopier

### PR DESCRIPTION
**Pull request by actioninja**
## Changelog
:cl:
add: Increased photocopier accuracy
/:cl:

### CODE REVIEW

**Comment by Tharcoonvagh** [comment]
wot

**Comment by optimumtact** [comment]
You know what has to be done

**Comment by Tharcoonvagh** [comment]
Jesus fucking christ

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -173,6 +173,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	///Bitflag that controls what in game ways something can select this species as a spawnable source, such as magic mirrors. See [mob defines][code/__DEFINES/mobs.dm] for possible sources.
 	var/changesource_flags = NONE
 
+	///For custom overrides for species ass images
+	var/icon/assimage
```
Comment (Jared-Fogle) [inline code/modules/mob/living/carbon/human/species.dm:177]:
```suggestion
	var/icon/ass_image
```

Comment (Jared-Fogle) [reply code/modules/mob/living/carbon/human/species.dm:177]: yes, im going to bikeshed the ass pr

**Comment by cacogen** [comment]
Soon there will be ass organs

**Comment by Pumpkinoe** [comment]
BORGS HAVE A QRCODE
this is the fucking best

**Comment by actioninja** [comment]
@optimumtact https://www.youtube.com/watch?v=9FgwKpemKmo

**Review by Mothblocks — APPROVED** [review]

**Comment by Iamgoofball** [comment]
scanned the QR code for the robot ass
TERS
:shipit:

**Comment by Zytolg** [comment]
Two and a half years later, and PR has only just now filled me with regret and my eyes with Bleach

**Comment by optimumtact** [comment]
this man photocopied a cats ass

**Pull request merged**